pkg/util/dynamichelper/dynamichelper.go

```diff
@@ -66,7 +81,25 @@ func New(log *logrus.Entry, restconfig *rest.Config) (Interface, error) {
 	return dh, nil
 }
 
-func (dh *dynamicHelper) EnsureDeleted(ctx context.Context, groupKind, namespace, name string) error {
+// GetResource uses the rest cluster connection to retrieves the GVR from
+// the requested namespace
+func (dh *DynamicHelper) GetResource(ctx context.Context, groupKind, namespace, name string) (kruntime.Object, error) {
+	gvr, err := dh.Resolve(groupKind, "")
```

Comment:
The empty string is following existing implementations. I'm not a fan of passing an empty string without more context.